packages/insured-bridge-relayer/src/CrossDomainFinalizer.ts

```diff
@@ -0,0 +1,91 @@
+import winston from "winston";
+
+import { createEtherscanLinkMarkdown, createFormatFunction, PublicNetworks } from "@uma/common";
+import { getAbi } from "@uma/contracts-node";
+import { InsuredBridgeL1Client, InsuredBridgeL2Client, GasEstimator } from "@uma/financial-templates-lib";
+
+export enum RelaySubmitType {
+  Slow,
+  SpeedUp,
+  Instant,
+  Ignore,
+}
+
+export class CrossDomainFinalizer {
+  constructor(
+    readonly logger: winston.Logger,
+    readonly gasEstimator: GasEstimator,
+    readonly l1Client: InsuredBridgeL1Client,
+    readonly l2Client: InsuredBridgeL2Client,
+    readonly account: string
+  ) {}
+  async checkForBridgeableL2TokensAndBridge() {
+    this.logger.debug({ at: "AcrossRelayer#CrossDomainFinalizer", message: "Checking bridgeable L2 tokens" });
```

Comment:
I think you should emit the L1 client's bridge admin address somewhere, perhaps in this log

Reply:
This is emitted in the start up log of the the bot in the here: https://github.com/UMAprotocol/protocol/blob/2fc8fe4237156e68a02bf15ff9f981eed0598137/packages/insured-bridge-relayer/src/index.ts#L31

do you want it again here?

Reply:
oh good point, nevermind